data/human/human missions.txt

```diff
@@ -5547,3 +5547,221 @@ mission "Quicksilver Mixup 1"
+				`	(Nevermind.)`
+					decline
+			`	"Yes, hello, how are you doing?" the man responds, shifting his head towards your direction. "I'm here waiting for someone to pick me up," he says in a voice that sounds like he expects it at any moment.`
+
+			label questions
+			choice
+				`	"Are they far away?"`
+					goto far
+					to display
+						not "Blind Man from Martini: label far"
+				`	"Who is it you're looking for?"`
+					goto looking
+					to display
+						has "Blind Man from Martini: label far"
+						not "Blind Man from Martini: label looking"
+				`	"Where is she?"`
+					goto somewhere
+					to display
+						has "Blind Man from Martini: label looking"
+				`	(Nevermind, he's just a blind man.)`
+					decline
+			
+			label far
+			action
+				set "Blind Man from Martini: label far"
+			`	"Well, I suppose you could say that they are far away, or they could be right here. I don't know, I can't see them." He taps his cane twice more.`
+				goto questions
+			
+			label looking
+			action
+				set "Blind Man from Martini: label looking"
+			`	"Someone to take me to my daughter, Anita." Yet again, his cane taps twice.`
+				goto questions
+
+			label somewhere
+			`	He looks over to his side, as if he feels reserved about something. "She's on <planet>, getting some experimental treatment from the University of New Sydney. I want to be by her side if anything goes wrong, to tell her that I'm there for her, that I love her, and that everything will be okay. She's been waiting for the treatment for so long now. And since nobody else seems to have arrived here yet, I hope you'll be the one to take me there."`
+			choice
+				`	"Of course, I'll take you to <planet>."`
+				`	"Sorry, I won't be able to."`
+					goto can't
+			`	The man musters a broad smile, says "Thank you all too much. Could you help me with the luggage? Of course, I can't see where it is..." and scans the floor with his cane for it. Later, while boarding, he tells you his name is Douglas, and he promises to you <payment> for transporting him.`
+				accept
+
+			label can't
+			`	"Then I will wait until another comes to help me." The blind man taps his cane once more, then returns to looking straight ahead.`
+				decline
+
+	on complete
+		log `Brought a blind man named Douglas to Chiron so he could visit his daughter, who will undergo experimental treatment at the University of New Sydney.`
+		payment 58000
+		conversation
+			`Throughout the trip, your blind passenger Douglas has been relatively quiet and has kept to himself. But when you land, he asks you again to help him bring down his luggage once more; it turns out you needn't have done so because an attendant from the planetary spaceport quickly comes to his side. They guide both of you to the spaceport's intracity transit complex; the bustling spaceport makes it a challenge to keep together, but all of you manage to reach it. From there, you hail a taxi while he arranges for the <payment> to be sent to you.`
```

Comment:
I just hadn't decided, to be honest.

Reply:
```suggestion
			`Throughout the trip, your blind passenger Douglas has been relatively quiet and has kept to himself. But when you land, he asks you again to help him bring down his luggage once more; it turns out you needn't have done so because an attendant from the planetary spaceport quickly comes to his side. He guides both of you to the spaceport's intracity transit complex; the bustling spaceport makes it a challenge to keep together, but all of you manage to reach it. From there, you hail a taxi while he arranges for the <payment> to be sent to you.`
```